llama_stack/providers/registry/post_training.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import List
+
+from llama_stack.distribution.datatypes import *  # noqa: F403
+
+
+META_REFERENCE_DEPS = [
```

Comment:
I think you can just inline them. The reason we had this constant elsewhere is merely because it was shared between two providers.